compute/admin_guide/agentless-scanning/agentless-scanning.adoc

```diff
@@ -41,16 +48,23 @@ This mode spins up temporary scanning instances in the account.
 
 ==== Hub Account Mode
 
-Scan hosts of the cloud account (the target) using another cloud account (the hub).
+Scan the workloads and container images of a cloud account (the target) using another cloud account (the hub).
 This mode spins up temporary scanning instances in the hub rather than in the target(s).
 
-. Onboard accounts with different permissions for hub account, which perform the scan, and target accounts, which the hub account scans. Follow the step-by-step instructions to https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/configure/configure-agentless-scanning[configure the permissions].  
+ifdef::compute_edition[]
+. Onboard accounts with different permissions for hub account, which perform the scan, and target accounts, which the hub account scans. Follow the step-by-step instructions to xref:./onboard-accounts/onboard-accounts.adoc[configure the permissions].
+endif::compute_edition[]
+
+ifdef::prisma_cloud[]
+. xref:./onboard-accounts/onboard-accounts.adoc[Onboard your cloud account to Prisma Cloud].
+endif::prisma_cloud[]
+
 . Prisma Cloud only spins up scanners in the dedicated hub account and attaches snapshots of instances from other accounts to the scanners in the hub account. 
 . Scanners send results to the Prisma Cloud Console
 . Scanners then get deleted along with the snapshots that Prisma Cloud creates.
 . Process repeats for periodic scans.
 
 === Next Steps
 
-Follow the instructions in xref:./onboard-accounts/onboard-accounts.adoc[Configure Agentless scanning document] and start scanning your AWS, Azure and GCP accounts for vulnerabilities and configuration risks with Agentless Scanning.
+Follow the instructions in xref:./onboard-accounts/onboard-accounts.adoc[Configure Agentless scanning document] and start scanning your AWS, Azure, GCP, and OCI accounts for vulnerabilities and configuration risks with agentless scanning.
```

Comment:
Not sure we need the Next Steps here because you have provided inline links to onboarding tasks.

Potentially, do you need to add this with the permissions section line 55 for on-prem?

I like the way you have it for PCEE xref:./onboard-accounts/onboard-accounts.adoc[Onboard your cloud account to Prisma Cloud].

instead of saying

Follow the instructions in xref:./onboard-accounts/onboard-accounts.adoc[Configure Agentless scanning **document**]